plugin/neoterm.vim

```diff
@@ -28,7 +28,7 @@ endfunction
 let g:neoterm_statusline = ''
 
 if !exists('g:neoterm_shell')
-  if exists('&shellcmdflag')
+  if exists('&shellcmdflag') && has('nvim')
```

Comment:
can you invert the logic here, please? Check for the platform first I think makes more sense.